packages/components/formatted-monetary-amount/stories/index.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import type { Story, Meta } from '@storybook/react';
+import { useArgs } from '@storybook/client-api';
+
+/**
+ * Internal dependencies
+ */
+import FormattedMonetaryAmount, { type FormattedMonetaryAmountProps } from '..';
+
+export default {
+	title: 'WooCommerce Blocks/@woocommerce-blocks-components/FormattedMonetaryAmount',
+	component: FormattedMonetaryAmount,
+	argTypes: {
+		className: {
+			control: 'text',
+			table: {
+				type: {
+					summary: 'string',
+				},
+			},
+		},
+		currency: {
+			table: {
+				type: {
+					summary: 'object',
+				},
+			},
+			control: 'object',
+			description:
+				'The currency object used to describe how the monetary amount should be displayed.',
+		},
+		displayType: {
+			table: {
+				type: {
+					summary: 'input | text',
+				},
+			},
+			control: 'select',
+			options: [ 'input', 'text' ],
+			defaultValue: 'text',
+			description:
+				'Whether this should be an input or just a text display.',
+		},
+		value: {
+			control: 'number',
+			table: {
+				type: {
+					summary: 'number',
+				},
+			},
+			defaultValue: 1234,
+			description:
+				'The raw value of the currency, it will be formatted depending on the props of this component.',
+		},
+		isAllowed: {
+			control: 'function',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+			if: { arg: 'displayType', eq: 'input' },
+			description:
+				'A checker function to validate the input value. If this function returns false, the onChange method will not get triggered and the input value will not change.',
+		},
+		allowNegative: {
+			control: 'boolean',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+			},
+			description: 'Whether negative numbers can be entered',
+		},
+		onValueChange: {
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+			action: 'changed',
+		},
+		style: {
+			control: 'object',
+			table: {
+				type: {
+					summary: 'object',
+				},
+			},
+		},
+	},
+} as Meta< FormattedMonetaryAmountProps >;
+
+const Template: Story< FormattedMonetaryAmountProps > = ( args ) => {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const [ _, updateArgs ] = useArgs();
```

Comment:
I would like to understand the usage of '_' here. I don't see it being used in the code.

Reply:
It's because we're destructuring an array and we don't want the first member, only the second. Since we can't just select the second (well, we can by doing `useArgs[1]` but that's not convention) we put an `_` to signify we're not using it.

Reply:
Thank you, that's TIL for me. 👍